aardwolf-models/Cargo.toml

```diff
@@ -20,7 +20,8 @@ strum = "0.25.0"
 strum_macros = "0.25.1"
 thiserror = "1.0"
 url = "2.1"
-dotenv="0.15"
+dotenvy ="0.15"
```

Comment:
**Suggestion:** The `dotenvy` package is added, but it's not clear if it's used in the code. If it's not used, consider removing it to reduce the project's dependencies.
```suggestion

```

Reply:
Dotenv is used as part of the config/environment setup.  Dotenvy is the *currently supported* fork.